utils/zerovec/benches/vzv.rs

```diff
@@ -158,5 +159,54 @@ fn serde_benches(c: &mut Criterion) {
     });
 }
 
+#[cfg(feature = "bench")]
+// Testing differences between operating on slices with precomputed/non-precomputed indexing info
+fn vzv_precompute_bench(c: &mut Criterion) {
+    let seed = 2021;
+    let (string_vec, seed) = random_alphanums(2..=20, 500, seed);
+    let (needles, _) = random_alphanums(2..=20, 10, seed);
+    let bytes: Vec<u8> = VarZeroVec::<str>::get_serializable_bytes(&string_vec).unwrap();
+    let vzv = VarZeroVec::<str>::parse_byte_slice(black_box(bytes.as_slice())).unwrap();
```

Comment:
Nit: You should use `VarZeroVec::from(&string_vec)` now.

Reply:
It doesn't work yet, I'll merge this since this has passing tests and push a patch to your branch

Reply:
Oh, it did work, I was fixing the wrong line. Oh well, the order of landing the PRs doesn't matter much :smile: